forui/lib/src/widgets/header/header.dart

```diff
@@ -1,30 +1,89 @@
+import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
 
 import 'package:forui/forui.dart';
+import 'package:forui/src/foundation/tappable.dart';
 
+part 'header_action.dart';
+
+/// A header widget.
+///
+/// Typically used on pages at the root of the navigation stack.
 final class FHeader extends StatelessWidget {
-  final Widget? left;
-  final Widget? right;
+  /// The title.
```

Comment:
```suggestion

  /// The title.
```

Reply:
I don't think there should be an extra space. Looking at the flutter codebase, this isn't the convention followed.

Reply:
Ah then, we should reformat the other files then.